set feeder motor limits and update motor control

Howard2503 opened the pull request.

Howard2503 committed:
set limits

LeoJhonSong reviewed (changes requested):
灏天投食和重置机械臂两个指令好像反了:joy:
另外三个电机的速度调一下吧, 现在这样感觉可能有点慢, 有可能从识别到橙色色块时开始转动机械臂, 而东西掉出来的时候小车已经走过橙色色块了, 那样的话很可能东西掉不到水里. 你试一下呢? 不行的话我想到几个方案:
1. 让机械臂转快一些
2. 橙色色块增长一些
3. 转动机械臂的同时将轮速降低

我们先不管穿模的事吧, 把步长就基本不穿模了, 我估计是不会实时仿真的, 那就步长为1也没事.

然后电机的名字还是改一下吧, 让写代码的时候更有辨识度些. 比如你的wheel3我看了模型才发现是右后轮而不是右前轮:joy:

LeoJhonSong committed:
feat(chassis): rename all motors, enhance wheel motors

increase maxVelocity, maxTorque and wheel motors in patio.wbt, so we are
able to speed up later

LeoJhonSong committed:
feat(chassis): increase density of chassis, reorganize robot node #127

increase density of body to density of metal, increase density of arm to
density of aluminium
reorganize the nodes under robot node

LeoJhonSong committed:
fix(chassis): modify arm node and velocity in chassis.py
